bin/unix/install-dependencies/archlinux.sh

```diff
@@ -0,0 +1,21 @@
+#/bin/bash
+#
+## 
+## Usage: archlinux
+## 
+## This script automates the building of ownCloud documentation on the ArchLinux platform.
+## While new, it should handle installing all of the required platform depencies, and afterwards
+## build the documentation. 
+##
+## Author: Matthew Setter <msetter@owncloud.com>
+##
+
+function install_dependencies()
+{
+  sudo pacman-key --refresh-keys && \ 
```

Comment:
Rather than chaining all the lines with `&&\` on every line, I would suggest running `set -e` just below the hash-bang. This will cause the script to abort on any command that fails. The resulting code is easier to read and maintain.